xbmc/cores/VideoPlayer/Edl.cpp

```diff
@@ -542,79 +526,48 @@ bool CEdl::ReadBeyondTV(const std::string& strMovie)
-  }
-  else if (fileItem.HasEPGInfoTag())
-  {
-    CLog::Log(LOGDEBUG, "%s - Reading Edl for EPG: %s", __FUNCTION__, fileItem.GetEPGInfoTag()->Title().c_str());
-    edl = fileItem.GetEPGInfoTag()->GetEdl();
-  }
-  else
-  {
-    CLog::Log(LOGERROR, "%s - Unknown file item type : %s", __FUNCTION__, CURL::GetRedacted(strMovie).c_str());
-    return false;
-  }
-
-  std::vector<PVR_EDL_ENTRY>::const_iterator it;
-  for (it = edl.begin(); it != edl.end(); ++it)
-  {
-    Cut cut;
-    cut.start = it->start;
-    cut.end = it->end;
-
-    switch (it->type)
+    switch (cut.action)
     {
-    case PVR_EDL_TYPE_CUT:
-      cut.action = Action::CUT;
-      break;
-    case PVR_EDL_TYPE_MUTE:
-      cut.action = Action::MUTE;
-      break;
-    case PVR_EDL_TYPE_SCENE:
-      if (!AddSceneMarker(cut.end))
-      {
-        CLog::Log(LOGWARNING, "%s - Error adding scene marker for pvr recording", __FUNCTION__);
-      }
-      continue;
-    case PVR_EDL_TYPE_COMBREAK:
-      cut.action = Action::COMM_BREAK;
-      break;
-    default:
-      CLog::Log(LOGINFO, "%s - Ignoring entry of unknown type: %d", __FUNCTION__, it->type);
-      continue;
-    }
+      case Action::CUT:
+      case Action::MUTE:
+      case Action::COMM_BREAK:
+        if (AddCut(cut))
+        {
+          CLog::Log(LOGDEBUG, "%s - Added break [%s - %s] found in PVR item for: %s.",
+            __FUNCTION__, MillisecondsToTimeString(cut.start).c_str(),
+            MillisecondsToTimeString(cut.end).c_str(), CURL::GetRedacted(fileItem.GetDynPath()).c_str());
+        }
+        else
+        {
+          CLog::Log(LOGERROR, "%s - Invalid break [%s - %s] found in PVR item for: %s. Continuing anyway.",
+            __FUNCTION__, MillisecondsToTimeString(cut.start).c_str(),
+            MillisecondsToTimeString(cut.end).c_str(), CURL::GetRedacted(fileItem.GetDynPath()).c_str());
+        }
+        break;
 
-    if (AddCut(cut))
-    {
-      CLog::Log(LOGDEBUG, "%s - Added break [%s - %s] found in PVRRecording for: %s.",
-        __FUNCTION__, MillisecondsToTimeString(cut.start).c_str(),
-        MillisecondsToTimeString(cut.end).c_str(), CURL::GetRedacted(strMovie).c_str());
-    }
-    else
-    {
-      CLog::Log(LOGERROR, "%s - Invalid break [%s - %s] found in PVRRecording for: %s. Continuing anyway.",
-        __FUNCTION__, MillisecondsToTimeString(cut.start).c_str(),
-        MillisecondsToTimeString(cut.end).c_str(), CURL::GetRedacted(strMovie).c_str());
+      case Action::SCENE:
+        if (!AddSceneMarker(cut.end))
+        {
+          CLog::Log(LOGWARNING, "%s - Error adding scene marker for PVR item", __FUNCTION__);
+        }
+        break;
+
+      default:
+        CLog::Log(LOGINFO, "%s - Ignoring entry of unknown cut action: %d", __FUNCTION__, static_cast<int>(cut.action));
+        break;
     }
   }
 
- return !edl.empty();
+  return !cutlist.empty();
 }
 
-bool CEdl::AddCut(Cut& cut)
+bool CEdl::AddCut(const Cut& NewCut)
```

Comment:
`newCut`

Reply:
I took the name from the header. Now name matches the name given in the header.

Reply:
Should it not start with lowercase `n`?

Reply:
yes, i will change that